client/stylesheets/_responsive.scss

```diff
@@ -155,12 +238,18 @@
     .homeCardLi {
     	width: 65%;
     }
+    .searchResultsCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardLi` should be written in lowercase with hyphens